plugins/modules/clickhouse_user.py

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, Aleksandr Vagachev (@aleksvagachev) <aleksvagachev@yandex.ru>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: clickhouse_user
+
+short_description: Creates or removes a ClickHouse user using the clickhouse-driver Client interface
+
+description:
+  - Creates or removes a ClickHouse user using the
+    L(clickhouse-driver,https://clickhouse-driver.readthedocs.io/en/latest) Client interface.
+  - The module can only create and delete users, without any additional parameters.
+    New features will be added in the future.
+
+attributes:
+  check_mode:
+    description: Supports check_mode.
+    support: full
+
+
+author:
+  - Aleksandr Vagachev (@aleksvagachev)
+
+extends_documentation_fragment:
+  - community.clickhouse.client_inst_opts
+
```

Comment:
```suggestion
version_added: '0.4.0'

```

Reply:
this is important ^. we don't use changelog fragments for new modules. ansibull-changelog will add an entry automatically by using this attribute